src/server/svr_func.c

```diff
@@ -3326,11 +3323,6 @@ set_entity_ct_sum_queued(job *pjob, pbs_queue *pque, enum batch_op op)
 		return rc;
 	}
 
-	if (op == INCR)
-		pjob->ji_entity_limit_set = 1;
-	else
-		pjob->ji_entity_limit_set = 0;
-
```

Comment:
removed `pjob->ji_entity_limit_set` as it created phantom usage when multiple queued limits are in force.